examples/timer/index.html

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <title>Yew • Timer</title>
-  </head>
 
-  <body></body>
-</html>
+<head>
+  <meta charset="utf-8" />
+  <title>Yew • Timer</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css" />
+</head>
+
+<body></body>
+
+</html>
```

Comment:
Why the hate for newlines? :(
```suggestion
</html>

```

Reply:
This is just the VSCode HTML formatter doing it's thing.